### PR TITLE
Mepts revert config removed concepts

### PR DESCRIPTION
**Pull request by psmatsinhe**

### CODE REVIEW

File: omod/src/main/resources/config.xml [hunk 1]
```diff
@@ -1465,6 +1474,186 @@
...
+		<property>eptsreports.unknownConceptUuid</property>
+		<defaultValue>e1d81d7e-1d5f-11e0-b929-000c29ad1d07</defaultValue>
+		<description>
+			UUID for the UNKNOWN concept
+		</description>
+	</globalProperty>
+
+	<!-- 23818 -->
+	<globalProperty>
+		<property>eptsreports.reasonForRequestingViralLoadConceptUuid</property>
```
Comment (cfaife) [inline omod/src/main/resources/config.xml:1488]: this concept is duplicated

**Review by cfaife — CHANGES_REQUESTED** [review]

File: omod/src/main/resources/config.xml [hunk 2]
```diff
@@ -1640,6 +1830,21 @@
...
+		<defaultValue>a97ab290-5b66-4755-bd49-d82f944cd093</defaultValue>
+		<description>
+			UUID for the Reason for requesting viral load concept
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>eptsreports.routineForRequestingViralLoadConceptUuid</property>
```
Comment (cfaife) [inline omod/src/main/resources/config.xml:1842]: this concept  is duplicated

File: omod/src/main/resources/config.xml [hunk 3]
```diff
@@ -1640,6 +1830,21 @@
 			UUID for the Absence of Health Provider in Health Unit concept
 		</description>
 	</globalProperty>
+	<globalProperty>
+		<property>eptsreports.reasonForRequestingViralLoadConceptUuid</property>
```
Comment (cfaife) [inline omod/src/main/resources/config.xml:1834]: this  metadata is duplicated

File: omod/src/main/resources/config.xml [hunk 4]
```diff
@@ -1465,6 +1474,186 @@
...
+		<globalProperty>
+		<property>eptsreports.quarterlyConceptUuid</property>
+		<defaultValue>f53848d5-3521-4cc8-ac72-d63adef281a1</defaultValue>
+		<description>
+			UUID for the TRIMESTRAL concept
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>eptsreports.completedConceptUuid</property>
```
Comment (cfaife) [inline omod/src/main/resources/config.xml:1549]: this  metadata is duplicated

File: omod/src/main/resources/config.xml [hunk 5]
```diff
@@ -1465,6 +1474,186 @@
...
+	<globalProperty>
+		<property>eptsreports.clinicalSuspicionConceptUuid</property>
+		<defaultValue>14ef99f0-88b2-4210-9e85-bfa002743728</defaultValue>
+		<description>
+			UUID for the Clinical suspicion concept
+		</description>
+	</globalProperty>
+	
+		<globalProperty>
+		<property>eptsreports.quarterlyConceptUuid</property>
```
Comment (cfaife) [inline omod/src/main/resources/config.xml:1541]: this  metadata is duplicated

File: omod/src/main/resources/config.xml [hunk 6]
```diff
@@ -1465,6 +1474,186 @@
...
+		<property>eptsreports.reasonForRequestingViralLoadConceptUuid</property>
+		<defaultValue>a97ab290-5b66-4755-bd49-d82f944cd093</defaultValue>
+		<description>
+			UUID for the Reason for requesting viral load concept
+		</description>
+	</globalProperty>
+
+	<!-- 23817 -->
+	<globalProperty>
+		<property>eptsreports.routineForRequestingViralLoadConceptUuid</property>
```
Comment (cfaife) [inline omod/src/main/resources/config.xml:1497]: this  metadata is duplicated

**Review by cfaife — CHANGES_REQUESTED** [review]

**Comment by ningosi** [comment]
Closing this PR because the fixes have been done in https://github.com/esaude/openmrs-module-eptsreports/pull/180

**Pull request closed without merging**